dev/test.ts

```diff
@@ -106,6 +106,7 @@ const env: Record<string, string> = {
   "TMP_DIR": tmpDir,
   "TIMER_MAX_TIMEOUT_MS": "30000",
   "NPM_CONFIG_REGISTRY": "http://localhost:4873",
+  "PATH": `${Deno.env.get("PATH")}:${join(projectDir, "target/debug")}`,
```

Comment:
You probably ought to prefix the build dir path since we want items in there to be resolved first.